docs/index.md

```diff
@@ -19,41 +19,59 @@
 
 # Introduction
 
-Uvicorn is a lightning-fast ASGI server, built on [uvloop][uvloop] and [httptools][httptools].
+Uvicorn is a lightning-fast ASGI server implementation, using [uvloop][uvloop] and [httptools][httptools].
 
 Until recently Python has lacked a minimal low-level server/application interface for
-asyncio frameworks. The [ASGI specification][asgi] fills this gap, and means we're now able to start building
-a common set of tooling usable across all asyncio frameworks.
+asyncio frameworks. The [ASGI specification][asgi] fills this gap, and means we're now able to
+start building a common set of tooling usable across all asyncio frameworks.
 
 ASGI should help enable an ecosystem of Python web frameworks that are highly competitive against Node
 and Go in terms of achieving high throughput in IO-bound contexts. It also provides support for HTTP/2 and
 WebSockets, which cannot be handled by WSGI.
 
 Uvicorn currently supports HTTP/1.1 and WebSockets. Support for HTTP/2 is planned.
 
----
-
 ## Quickstart
 
-Requirements: Python 3.5, 3.6, 3.7, 3.8
-
 Install using `pip`:
 
-```
+```shell
 $ pip install uvicorn
```

Comment:
these changes are from the README.md which seems more up to date - if I get any CR comments on this I'll sync them back to the README.md